features/shop/cart_with_multiple_tax_rates.feature

```diff
@@ -0,0 +1,30 @@
+@ui-cart
+Feature: Cart with multiple tax rates
+    In order to order products with different tax rate
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And store has "EU VAT" tax rate of 23% for "Clothes"
```

Comment:
is this feature not supposed to be zone-based? Might be worth explaining that somewhere if not.